projects/examples/src/components/datagrid/datagrid-height.example.component.ts

```diff
@@ -16,10 +16,24 @@ interface Data {
 @Component({
     selector: 'vcd-datagrid-height-example',
     template: `
-        <div style="height:300px">
-            <vcd-datagrid [gridData]="gridData" (gridRefresh)="refresh($event)" [columns]="columns"> </vcd-datagrid>
+        <div
+            style="height:300px;
+                    flex-direction: column;
```

Comment:
This looks misaligned, can't we just use CSS instead?

Reply:
We should explain in the documentation that this requires that the parent use `display: flex; flex-direction: column`